crates/aggregator/src/rpc_server.rs

```diff
@@ -7,7 +7,7 @@ use tarpc::{context::Context, ServerError};
 use tracing::info;
 
 #[tarpc::service]
-/// This is the service definition. It defines one RPC, [`process_signed_task_response`].
+/// This is the service definition. It defines one RPC, `process_signed_task_response`.
```

Comment:
I think this can be linked correctly by doing:

```suggestion
/// This is the service definition. It defines one RPC, [`process_signed_task_response`](ProcessSignedTaskResponse::process_signed_task_response).
```